code/modules/merchant/merchant_programs.dm

```diff
@@ -9,8 +9,9 @@
 	size = 12
 	usage_flags = PROGRAM_CONSOLE
 	requires_access_to_run = PROGRAM_ACCESS_LIST_ONE
-	required_access_run = list(ACCESS_MERCHANT)
-	required_access_download = list(ACCESS_MERCHANT)
+	requires_access_to_download = PROGRAM_ACCESS_LIST_ONE
+	required_access_run = list(ACCESS_MERCHANT, ACCESS_GOLDEN_DEEP, ACCESS_GOLDEN_DEEP_OWNED, ACCESS_MERCHANTS_GUILD, ACCESS_NKA)
+	required_access_download = list(ACCESS_MERCHANT, ACCESS_GOLDEN_DEEP, ACCESS_GOLDEN_DEEP_OWNED, ACCESS_MERCHANTS_GUILD, ACCESS_NKA)
```

Comment:
why does it need access reqs at all? At this point may as well remove it so it can be used on any away site or offship, instead of just adding more and more accesses to one program used by everyone